pom.xml

```diff
@@ -214,9 +214,6 @@
             <resource>
                 <directory>src/main/resources</directory>
             </resource>
-            <resource>
-                <directory>target/metamodel</directory>
```

Comment:
Ah very annoying. Yet I hope it didn't break the outputted jar. It needs it's resources :wink: